notebooks/Example_Fitting_a_Linear_Model.ipynb

```diff
@@ -39,14 +39,12 @@
    "outputs": [
```

Comment:
<p>Please put the comments into the text cell above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/JuBiotech/calibr8/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>